### PR TITLE
Add tests for the ValidatorPropertyMetadataFactory class

### PR DESCRIPTION
**Pull request by meyerbaptiste**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

Maybe we can make our "advent" tests calendar and get 100% coverage for the end of the year...

### CODE REVIEW

**Comment by Simperfit** [comment]
Could you rebase please ?

**Comment by soyuka** [comment]
@Simperfit No conflicts, I think the "Update branch" button will work fine no?

**Comment by Simperfit** [comment]
@soyuka I know but it adds a commit to the PR no ?

**Comment by Simperfit** [comment]
Thanks @meyerbaptiste

File: tests/Fixtures/DummyEntityValidator.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+namespace ApiPlatform\Core\Tests\Fixtures;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Dummy Entity Validator.
+ *
+ * @author Baptiste Meyer <baptiste.meyer@gmail.com>
+ */
+class DummyEntityValidator
```
Comment (teohhanhui) [inline tests/Fixtures/DummyEntityValidator.php]: The class name is wrong. This is not a validator.

**Pull request merged**